### ISSUE TITLE
Windows for Workgroups 3.11 hangs immediately

### ISSUE THREAD

**Issue by ShanuJackal** [jeffpar/pcjs#66]
Does not load at all past that screen with random text characters in colors (I don't know what to call that screen)

**Comment by qodeos** [comment]
Yes. If I run it I get general protection fault in KRNL386.EXE , use a different x86 vm software

**Comment by qodeos** [comment]
That screen is the vga initial screen for video testing (csm post test)
